ci-operator/step-registry/windows/e2e/operator/test/upgrade/install/windows-e2e-operator-test-upgrade-install-ref.yaml

```diff
@@ -0,0 +1,17 @@
+ref:
+  as: windows-e2e-operator-test-upgrade-install
+  cli: latest
+  from: operator-sdk
+  commands: windows-e2e-operator-test-upgrade-install-commands.sh
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 400Mi
+  dependencies:
+    - env: PREVIOUS_BUNDLE
+      name: previous-bundle
+    - env: PREVIOUS_OPERATOR
+      name: previous-wmco
```

Comment:
previous -> initial

previous made sense in the original context as it was defined in a specific release's test config.
i.e. in the context of release-4.14 the previous-wmco is coming from release-4.14

By making this a general test step, these images/variable names now exist in a vacuum and need to be more specific.

Reply:
> i.e. in the context of release-4.14 the previous-wmco is coming from release-4.14

not following here;

Reply:
I meant
` i.e. in the context of release-4.14 the previous-wmco is coming from release-4.13`

Reply:
Thinking more about this I'd say `previous-bundle` should be renamed to `initial-wmco-bundle`

By putting this in the step-registry steps, you have taken on the additional scope of adding test steps that can be called from any repo not just WMCO.
These steps have been given dependencies that require specific images to be added to the job.
As the PR currently stands, lets say the upgrade tests need to be ran against `ovn-kubernetes` PRs.
In the config for `ovn-kubernetes` they would need to add an image `previous-bundle`. That doesn't make sense there, and depending on the repo it could cause confusion and a potential conflict.

That is why its important to do this.

Reply:
Thanks for sharing the high-level overview.